### PR TITLE
add a single apt-get command for linux

### PR DESCRIPTION
**Pull request by SimonDanisch**
This makes it much easier to install dependencies in one go!

### CODE REVIEW

**Comment by waldyrious** [comment]
While you're at it, perhaps also change the `-` to a `—` in the link about the `awk` dependency, to match the other ones (just noticed this while looking at the diff).

**Comment by Ismael-VC** [comment]
We only need either curl or wget, but not both. Thanks for adding this, I always forget.

File: README.md [hunk 1]
```diff
@@ -287,11 +287,16 @@ Building Julia requires that the following software be installed:
 - **[perl]**                    — preprocessing of header files of libraries.
 - **[wget]**, **[curl]**, or **[fetch]** (FreeBSD) — to automatically download external libraries.
 - **[m4]**                      — needed to build GMP.
-- **[awk]**                     - helper tool for Makefiles.
+- **[awk]**                     — helper tool for Makefiles.
```
Comment (garrison) [inline README.md]: Should also add the link for awk (below) while you are at it.

File: README.md [hunk 2]
```diff
@@ -287,11 +287,16 @@ Building Julia requires that the following software be installed:
...
 - **[patch]**                   — for modifying source code.
 - **[cmake]** (>= 3.4.3)        — needed to build `libgit2`.
 - **[pkg-config]**              — needed to build `libgit2` correctly, especially for proxy support.
 
+On e.g. Ubuntu you can easily install them with `apt-get`:
```
Comment (garrison) [inline README.md]: Could instead say "On Debian-based distributions (e.g. Ubuntu), [...]", but this is up to you.

Comment (SalchiPapa) [reply README.md]: Sometimes the packages names change even among related distro families, if the command is portable, then I agree, else another line for debian could be added and some other popular distro choices?

**Comment by SimonDanisch** [comment]
*updated*

**Review by garrison — APPROVED** [review]

**Comment by SimonDanisch** [comment]
ping!

**Comment by ViralBShah** [comment]
Can you add the link for `awk` and change the dash to match the others?

**Comment by SimonDanisch** [comment]
i did!

**Comment by ViralBShah** [comment]
Sorry - it looked like an optical illusion!

**Pull request merged**